pkg/types/installconfig.go

```diff
@@ -37,6 +38,7 @@ var (
 		gcp.Name,
 		openstack.Name,
 		ovirt.Name,
+		powervs.Name,
```

Comment:
Should we move to hidden platforms initially?

Reply:
yep